easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1_bamtools_as_dependency.patch

```diff
@@ -0,0 +1,34 @@
+diff -ruN rmats-turbo-4.1.1_orig/rMATS_pipeline/setup.py rmats-turbo-4.1.1/rMATS_pipeline/setup.py
```

Comment:
please add short patch description on top + mention author or where the patch comes from